### ISSUE TITLE
No suitable image found

### ISSUE THREAD

**Issue by fean** [FrontierPsychiatrist/node-spotify#87]
Hi y'all,

I'm having a bit of a problem.
It looks like the module finds the image but it's not for 32-bits or the other way around.

```Error: dlopen(/Users/leonardbreitkopf/Documents/Git/Spotty/Spotty.Player.OSX/MusicX.app/Contents/MacOS/data/nodespotify.node, 1): no suitable image found.  Did find:
	/Users/leonardbreitkopf/Documents/Git/Spotty/Spotty.Player.OSX/MusicX.app/Contents/MacOS/data/nodespotify.node: mach-o, but wrong architecture```

Could one of you guys help me out? Do I need a different version of node or some other module?

This I've done:

1. Installed the libspotify.framework version 12.1.51 in /Library/Frameworks

Setup:
Macbook 15" mid 2014 @ OSX Yosemite
Trying to use it in combination with AppJS.

**Comment by FrontierPsychiatrist** [comment]
Did you download node-spotify from somewhere or build it from source?

**Comment by fean** [comment]
I downloaded the release from the project website. But after that didn't work, I built it from source with the `-framework libspotify.framework` switch set in binding.gyp.That worked, it's running like a charm now. 

We could do this because we were only going to use the nodeJS release on OSX, on Windows we already have a native version of the client running. 

One thing that might need some optimizing is the pause function. It doesn't pause the playback right away. I have no experience at all with low-level API's outside of the Windows environment so I have no idea how hard that is.
